modules/cert-manager-enable-metrics.adoc

```diff
@@ -26,11 +26,43 @@ $ oc label namespace cert-manager openshift.io/cluster-monitoring=true
 
 . Create a service monitor:
 
-.. Create a YAML file that defines the `ServiceMonitor` object:
+.. Create a YAML file that defines the `Role`, `RoleBinding` and `ServiceMonitor` objects:
 +
-.Example `service-monitor.yaml` file
+.Example `monitoring.yaml` file
```

Comment:
Because changing the file name, we also need to update below: https://github.com/openshift/openshift-docs/blob/84928811c84c46a0778035636982fee94c2b4b6b/modules/cert-manager-enable-metrics.adoc?plain=1#L92